DESCRIPTION

```diff
@@ -1,6 +1,6 @@
 Package: Rcpp
 Title: Seamless R and C++ Integration
-Version: 0.12.18.2
+Version: 0.12.18.3
```

Comment:
.3 is already used in the other open PR.  Can you maybe cherry-pick over here or accomodate otherwise?